.remarkrc

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Nice! How can we use it, is there a documentation on how to write the documentation? :)